README.md

```diff
@@ -51,6 +53,20 @@ Provide, at a minimum, the required driver options in your `.kitchen.yml` file:
 The `image_ref` and `flavor_ref` options can be specified as an exact id,
 an exact name, or as a regular expression matching the name of the image or flavor.
 
+With the update to test-kitchen 1.4 there is now a `transport` layer that is modular.
```

Comment:
I'd suggest making this a little clearer...

Test Kitchen 1.4 supports multiple transports, and transports can be configure globally:

```yaml
transport:
  username: ubuntu
  password: mysecretpassword
```

... or per-platform:

```yaml
platforms:
  name: ubuntu-14.04
  transport:
    password: myrootpassword
  name: windows-2012r2
    password: myadministratorpassword
```